src/prefect/tasks/google/storage.py

```diff
@@ -256,3 +256,113 @@ def run(
         gcs_blob = self._get_blob(bucket, blob, encryption_key_secret)
         gcs_blob.upload_from_string(data)
         return gcs_blob.name
+
+
+class GCSCopy(Task):
+    """
+    Task template for copying data from one Google Cloud Storage blob to another, without
```

Comment:
```suggestion
    Task template for copying data from one Google Cloud Storage bucket to another, without
```

Reply:
_technically_ blob is correct but bucket feels correct to me. While it is duplicating a blob it's really copying a blob from one bucket to another.